### ISSUE TITLE
Empty Window 

### ISSUE THREAD

**Issue by khalilhas** [butterproject/butter-desktop#664]
### Description

After clicking "accept" for the term of use,  it shows an empty window with no Films or TV shows just an empty window with "Buffer" written on top and the "x" button , minimizing etc.
I don't have any error running the commands "gulp build" nor "gulp run", I did all the steps included in the read me file.
 
- Windows 8 Pro

**Comment by jmpace** [comment]
I'm having the same problem, on MacOS 10.11.6.

**Comment by vankasteelj** [comment]
What branch are you using?

**Comment by jmpace** [comment]
Master.

**Comment by vankasteelj** [comment]
Yep, I can reproduce. @xaiki and @ChrisAlderson broke something with the new providers. 

(pssst: I told you to use versionning for providers, xaiki)

**Comment by vankasteelj** [comment]
Meanwhile, you should unzip and drop all the folders inside `butter-desktop/node_modules/`

[fix-providers.zip](https://github.com/butterproject/butter-desktop/files/1293885/fix-providers.zip)

**Comment by jmpace** [comment]
For what it's worth, replacing the `node_modules` folders with the ones you provided in that zip works! First time using Butter... seeing substantially less content than in Popcorn Time. Not sure if that's expected.
